app/models/uploaded_file.rb

```diff
@@ -0,0 +1,14 @@
+class UploadedFile < ApplicationRecord
+
+  belongs_to :membership_application
+
+  has_attached_file :actual_file
+
+  validates_attachment :actual_file, presence: true,
+                       content_type: { content_type: ['image/jpeg', 'image/gif', 'image/png',
+                                                     'text/plain',
+                                                     'text/rtf',
+                                                     'application/pdf',
+                                                     'application/msword'],
+                                       message: 'Sorry, this is not a file type you can upload.'}
```

Comment:
I think MIME type `application/msword` is limited to the old ".doc" extension.  

We will need to confirm support for the ".docx" extension as well (I assume).   The MIME type that I have used for that is `application/vnd.openxmlformats-officedocument.wordprocessingml.document` (yup, pretty ugly).